utilities/opentelemetry_instrumentation_http/src/opentelemetry_instrumentation_http.erl

```diff
@@ -62,6 +65,201 @@ attribute_name(request, HeaderName) ->
+code(not_implemented) -> 501;
+code(bad_gateway) -> 502;
+code(service_unavailable) -> 503;
+code(gateway_timeout) -> 504;
+code(http_version_not_supported) -> 505;
+code(variant_also_negotiates) -> 506;
+code(insufficient_storage) -> 507;
+code(loop_detected) -> 508;
+code(not_extended) -> 510;
+code(network_authentication_required) -> 511.
+
+-spec code_to_atom(integer()) -> atom().
+code_to_atom(100) -> continue;
+code_to_atom(101) -> switching_protocols;
+code_to_atom(102) -> processing;
+code_to_atom(103) -> early_hints;
+code_to_atom(200) -> ok;
+code_to_atom(201) -> created;
+code_to_atom(202) -> accepted;
+code_to_atom(203) -> non_authoritative_information;
+code_to_atom(204) -> no_content;
+code_to_atom(205) -> reset_content;
+code_to_atom(206) -> partial_content;
+code_to_atom(207) -> multi_status;
+code_to_atom(208) -> already_reported;
+code_to_atom(226) -> im_used;
+code_to_atom(300) -> multiple_choices;
+code_to_atom(301) -> moved_permanently;
+code_to_atom(302) -> found;
+code_to_atom(303) -> see_other;
+code_to_atom(304) -> not_modified;
+code_to_atom(305) -> use_proxy;
+code_to_atom(306) -> switch_proxy;
+code_to_atom(307) -> temporary_redirect;
+code_to_atom(308) -> permanent_redirect;
+code_to_atom(400) -> bad_request;
+code_to_atom(401) -> unauthorized;
+code_to_atom(402) -> payment_required;
+code_to_atom(403) -> forbidden;
+code_to_atom(404) -> not_found;
+code_to_atom(405) -> method_not_allowed;
+code_to_atom(406) -> not_acceptable;
+code_to_atom(407) -> proxy_authentication_required;
+code_to_atom(408) -> request_timeout;
+code_to_atom(409) -> conflict;
+code_to_atom(410) -> gone;
+code_to_atom(411) -> length_required;
+code_to_atom(412) -> precondition_failed;
+code_to_atom(413) -> request_entity_too_large;
+code_to_atom(414) -> request_uri_too_long;
+code_to_atom(415) -> unsupported_media_type;
+code_to_atom(416) -> requested_range_not_satisfiable;
+code_to_atom(417) -> expectation_failed;
+code_to_atom(418) -> im_a_teapot;
+code_to_atom(421) -> misdirected_request;
+code_to_atom(422) -> unprocessable_entity;
+code_to_atom(423) -> locked;
+code_to_atom(424) -> failed_dependency;
+code_to_atom(425) -> too_early;
+code_to_atom(426) -> upgrade_required;
+code_to_atom(428) -> precondition_required;
+code_to_atom(429) -> too_many_requests;
+code_to_atom(431) -> request_header_fields_too_large;
+code_to_atom(451) -> unavailable_for_legal_reasons;
+code_to_atom(500) -> internal_server_error;
+code_to_atom(501) -> not_implemented;
+code_to_atom(502) -> bad_gateway;
+code_to_atom(503) -> service_unavailable;
+code_to_atom(504) -> gateway_timeout;
+code_to_atom(505) -> http_version_not_supported;
+code_to_atom(506) -> variant_also_negotiates;
+code_to_atom(507) -> insufficient_storage;
+code_to_atom(508) -> loop_detected;
+code_to_atom(510) -> not_extended;
+code_to_atom(511) -> network_authentication_required.
+
+-spec code_to_phrase(integer()) -> binary().
+code_to_phrase(100) -> <<"Continue">>;
```

Comment:
These values can all be atoms. Avoid strings for any value known at compile time. That removes the need for the `code_to_atom/1`.